connector_magento/tests/test_export_invoice.py

```diff
@@ -0,0 +1,181 @@
+# Copyright 2013-2019 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html)
+
+from odoo import fields
+from odoo.tests.common import Form
+
+from .common import MagentoSyncTestCase, recorder
+
+
+class TestExportInvoice(MagentoSyncTestCase):
+    """ Test the export of an invoice to Magento """
+
+    def setUp(self):
+        super().setUp()
+        self.sale_binding_model = self.env["magento.sale.order"]
+        self.payment_mode = self.env["account.payment.mode"].search(
+            [("name", "=", "checkmo")], limit=1
+        )
+        self.pay_account = self.env["account.account"].search(
+            [("code", "=", "101501")], limit=1
+        )
```

Comment:
Same doubts here as above... What if the needed objects are not in the environment? You should prevent that and create the ones you need on the test...